docs/RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release process and check offs (client only)
+______
+since there is no server component at this time
+
+## Checkoffs
+
+## Product checks
```

Comment:
I assume that content review is part of WHO checkoff, since WHO is the ultimate arbiter of content, right? Do we need a separate checkoff here?

Reply:
I have added the legal bits

Reply:
@shankari I think we should do an internal content review before WHO handoff to catch basic things. WHO resources are limited, so we should make sure we're showing them something as complete as possible

Reply:
@hspinks do you have a sense of who will do the internal content review? Overall, I think that the code testing story is pretty good, but I am not sure there are sufficient teams/process for the non-automated checks.

e.g. @viviancromwell said on slack:
> Outside these two things, there is a need for running user testing  from internal release, beta to public release. That is a huge task and honestly I am not sure there is a team in place for that.

I am not sure whether issues like that should be raised in here or in the handover documents that @advayDev1 plans to work on.

Reply:
For now, I added a link to https://github.com/WorldHealthOrganization/app/issues/327 which tracks the plans for user testing.

Reply:
@hspinks I think it would be useful for you to outline what content needs to be checked. 

- The UX <-> mockup matching is checked as part of the design checkoff,
- the legal, privacy and security parts are checked off as part of the compliance checkoff 
- and the store assets are checked as part of the store asset checkoff

What other content do you have in mind?

Reply:
I have added a placeholder for content checking and filed an issue, but would prefer not leaving behind dangling issues if I can avoid it.

Reply:
@shankari as I think more about this, the content check is actually not going to be relevant from an app release perspective anyway, since content will be able to be updated dynamically from the servers. I'll remove this is a blocker to my 👍 on this PR, and this will be more generally a topic of discussion as we think about who is responsible for creating content for the app